recipes/sisl/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "sisl" %}
+{% set version = "0.9.2" %}
+{% set sha256 = "29f995077180dc43d30b6c55a2e933182912661d8ed8cfaf3f4c4ac40306fe91" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/zerothi/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [not (py27 or py>34)]
```

Comment:
Having multiple `skip` keys like this slightly confuses some of the tooling. It's better to do something like:
```
skip:
  - True  # [win]
  - True  # [not (py27 or py>34)]
```
– though note that conda-forge now only builds on 2.7, 3.5, and 3.6, so the latter `skip` statement isn't actually necessary.

Reply:
thanks, will fix!